<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="RobotController">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot Controller</string>
    </property>
    <widget class="QPushButton" name="forward_move">
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>72</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>forward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="left_move">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>85</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="right_move">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>111</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backward_move">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>backward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_parameter">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(0, 255, 0)</string>
     </property>
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_engine">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>111</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>速度(0~1)：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="ratio_speed">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>221</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>方位(-180～180)：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="advanced_angle">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Monitor</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="camera1">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 255, 127)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="camera2">
        <property name="styleSheet">
         <string notr="true">background:rgb(249, 75, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="camera3">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 170, 127)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="camera4">
        <property name="styleSheet">
         <string notr="true">background:rgb(85, 85, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>341</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="open_camera">
        <property name="text">
         <string>open camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_camera">
        <property name="text">
         <string>close camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="screen_shot">
        <property name="text">
         <string>screen shot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
